addons/mail/models/res_config_settings.py

```diff
@@ -38,6 +38,7 @@ class ResConfigSettings(models.TransientModel):
     )
     email_primary_color = fields.Char(related='company_id.email_primary_color', readonly=False)
     email_secondary_color = fields.Char(related='company_id.email_secondary_color', readonly=False)
+    module_whatsapp = fields.Boolean('Whatsapp Integration')
```

Comment:
Use-facing strings should use the proper name of the app, which is written `WhatsApp`.